src/worker.cc

```diff
@@ -179,6 +180,8 @@ const std::unordered_map<unsigned, std::string> OSRM_ERRORS_CODES{
      R"({"code":"InvalidValue","message":"The successfully parsed query parameters are invalid."})"},
     {142,
      R"({"code":"InvalidValue","message":"The successfully parsed query parameters are invalid."})"},
+    {143,
+     R"({"code":"InvalidValue","message":"The successfully parsed query parameters are invalid."})"},
```

Comment:
Can this error message specify what was invalid about the query?

Reply:
> Once we can confirm that is the case and those test cases are failing, we can disable them and get thsi shipped with the idea that we will do smoe more work. specifically i think we should make starting and ending in a closure work like starting/ending in a not_thru area, where you are allowed to route inside of it until you leave it. as long as your predicessory label is marked that way you can keep expanding in there but as soon as the predecessor is not marked that way you can no longer expand edges like that. this will require us to know whether or not an edge is closed outside of the allowed method though which will be quite annoying...

I get how continuing to expand on consecutive closures would work when doing bidir a-star (since we start expansion from both ends). But what would the behavior be for  the unidir a-star variants if there are multiple consecutive closures leading to the destination? When expanding from an intermediate open edge, we would avoid the closure, even though it might lead to the destination. If so, would it fail to find the route (what would happen if this was a not-thru edge instead)?

Reply:
i cant remember if the forward/reverse only implement handle this, i have to check the code to refresh my memory on that

Reply:
> Can this error message specify what was invalid about the query?

The specific error message is described in the [new internal error code](https://github.com/valhalla/valhalla/pull/2685/files#diff-7ab49622366d941710bed95da599b77e3bd3209104867f93e169fb9843dd4a8fR50). I think the error messages here are kept to match OSRM, which has this message for most error codes (@kevinkreiser CMIIW).